tool-openssl/x509.cc

```diff
@@ -250,6 +277,64 @@ bool X509Tool(const args_list_t &args) {
       }
     }
 
+    if(text) {
+      X509_print(output_bio.get(), x509.get());
+    }
+
+    if (subject) {
+      X509_NAME *subject_name = X509_get_subject_name(x509.get());
+      if (!subject_name) {
+        fprintf(stderr, "Error: unable to obtain subject from certificate\n");
+        return false;
+      }
+
+      BIO_printf(output_bio.get(), "subject=");
+      X509_NAME_print_ex(output_bio.get(), subject_name, 0, XN_FLAG_ONELINE);
+      BIO_printf(output_bio.get(), "\n");
+    }
+
+    if (fingerprint) {
+      int j;
+      unsigned int n;
```

Comment:
Please use more descriptive names, or move temp/index variables into a loop to make it clear.

Reply:
Noted